uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1002.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_Upgrader_Step_1002 {
+
+  /**
+   * Creates the 'other' sickness reason option value.
+   *
+   * @return bool
+   */
+  public function upgrade_1002() {
+    try {
+      $result = civicrm_api3('OptionValue', 'get', [
+        'sequential' => 1,
```

Comment:
minor, but you don't really need sequential here if just checking count. Indeed, you could just use the `getcount` method, like

```
$result = civicrm_api3('OptionValue', 'getcount', array(
  'name' => "other",
  'option_group_id' => "hrleaveandabsences_sickness_reason",
));
```